modules/test/playwright/tests/document-library-web/main/fileEntry.spec.ts

```diff
@@ -1076,3 +1076,54 @@ test(
 		).not.toBeVisible();
 	}
 );
+
+test(
+	'For documents in Pending status, both Submit for Workflow and Checkout actions must be disabled',
+	{tag: '@LPD-73328'},
+	async ({
+		documentLibraryEditFilePage,
+		documentLibraryEditFolderPage,
+		documentLibraryPage,
+		page,
+		site,
+	}) => {
+		const folderName = `PendingTestFolder-${getRandomString()}`;
+		const documentName = `PendingWorkflowDoc-${getRandomString()}.txt`;
+
+		await test.step('Set up folder and view', async () => {
+			await documentLibraryPage.goto(site.friendlyUrlPath);
+			await documentLibraryPage.changeView('Card');
+			await documentLibraryPage.goToCreateNewFolder();
+			await documentLibraryEditFolderPage.fillTitle(folderName);
+			await documentLibraryEditFilePage.saveButton.click();
+		});
+
+		await test.step('Update folder workflow to Single Approver', async () => {
+			await documentLibraryPage.goToEditFolder(folderName);
```

Comment:
Similar to what I mentioned above, for clarity and reusability I would create a function in `documentLibraryEditFolderPage` called `setWorkflow()` that do this.